src/sortByPath.js

```diff
@@ -0,0 +1,54 @@
+import { ascend, identity, map, path, pipe, sortWith, useWith } from 'ramda';
+
+const pathToAscendSort = pipe(path, ascend);
+const mapPathsToAscendSort = map(pathToAscendSort);
+
+/**
+ * Sort a list of objects by a list of paths (if first prop value is equivalent, sort by second, etc).
+ *
+ * @func sortByPath
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.29.0|v2.29.0}
```

Comment:
Not sure about the version

Reply:
Don't worry about it,  I'll change it before the merge.